pkg/nfs/nodeserver.go

```diff
@@ -82,25 +83,45 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	return &csi.NodePublishVolumeResponse{}, nil
 }
 
+func (ns *nodeServer) IsNotMountPoint(path string) (bool, error) {
+	mtab, err := ns.mounter.List()
+	if err != nil {
+		return false, err
+	}
+
+	for _, mnt := range mtab {
+		// This is how a directory deleted on the NFS server looks like
+		deletedDir := fmt.Sprintf("%s\\040(deleted)", mnt.Path)
```

Comment:
When pod's IP address changes - does this "deleted" text appears in mount table? what happens if NFS server is unavailable?

Reply:
This text appears when the mounted directory is deleted on the NFS server. It indicates a stale NFS handle.

It's copied from in-tree code, https://github.com/kubernetes/kubernetes/blob/449810c7858cc84e98ff0c859f863a40b56710dc/vendor/k8s.io/utils/mount/mount_helper_unix.go#L155